client/src/components/Document.tsx

```diff
@@ -0,0 +1,153 @@
+                            <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'>1: {scale}</i>
+                        </div>
+
+                        <div className="flex items-center justify-between mb-2 space-x-2">
+                            <i className="text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Issuance Date:</i>
+                            <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'> {issuanceDate?.toString()}</i>
+                        </div>
+
+                        <div className="flex items-center justify-between mb-2 space-x-2">
+                            <i className="text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Type:</i>
+                            <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'>{type} </i>
+                        </div>
+
+                        <div className="flex items-center justify-between mb-2 space-x-2">
+                            <i className="text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Language:</i>
+                            <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'>{language}</i>
+                        </div>
+
+                        <div className="flex items-center justify-between mb-2 space-x-2">
+                            <i className="text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Pages:</i>
+                            <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'> {pages} </i>
+                        </div>
+
+                    </div>
+                    <div className='hidden lg:flex flex-col items-start col space-y-2 w-1/2'>
+                        <i className="text-md font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Description:</i>
+                        <i className='text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong'> {description}  </i>
+                    </div>
+                    <i onClick={() => setShare(true)}><RiShareLine className="text-2xl text-tremor-content-strong dark:text-dark-tremor-content-strong" /></i>
+                    <Dialog open={share} onClose={() => setShare(false)} static={true}>
+                        <DialogPanel>
+                            <h3 className="text-lg font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Share "{title}"</h3>
+                            <p className="mt-2 leading-6 text-tremor-default text-tremor-content dark:text-dark-tremor-content">
+                                Share the link of the document with your friends and colleagues.
```

Comment:
remove friends

Reply:
So, also colleagues... it's not specified...